introduction.md

```diff
@@ -2,3 +2,9 @@
 Not sure how to edit and save in vi anymore, which sucks. I used to be a master of vi, now I haven't had a reason to use it in probably fifteen, sixteen years.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?